core/src/main/scala-3/spire/syntax/Ops.scala

```diff
@@ -0,0 +1,17 @@
+package spire
+package syntax
+
+import spire.algebra._
+import spire.algebra.partial._
+
+final class GroupoidCommonOps[A](lhs: A)(implicit ev: Groupoid[A]):
+  def inverse(): A = ev.inverse(lhs)
```

Comment:
Are the parenthesis necessary? I don't believe the method is supposed to be side-effectful (Scala idiom).

Reply:
I think I mentioned on the overall description that there is no uniform style with respect to empty or no parenthesis style in spire, and it causes some warnings.
I decided to keep them as they are new but I have a next PR planned that  will unify all of them in one go removing the parenthesis everywhere. Note that in several cases the old macro required them but I have fix for that

Reply:
We only need to be careful, removing parenthesis might be a source incompatible change for library users. We'll have to see.